x/btcstaking/client/cli/query_params.go

```diff
@@ -35,3 +35,29 @@ func CmdQueryParams() *cobra.Command {
 	return cmd
 }
 
+func CmdQueryParamsVersions() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "params-versions",
+		Short: "shows all the parameters of the module with version",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			res, err := queryClient.ParamsVersions(cmd.Context(), &types.QueryParamsVersionsRequest{})
```

Comment:
Pagination missing on request. You can get it from the flags

```suggestion
                        pageReq, err := client.ReadPageRequest(cmd.Flags())
			if err != nil {
				return err
			}
			res, err := queryClient.ParamsVersions(cmd.Context(), &types.QueryParamsVersionsRequest{
			     Pagination: pageReq,
			})
```

Reply:
good catch